lib/context.js

```diff
@@ -370,31 +380,42 @@ api.createTermDefinition = (activeCtx, localCtx, term, defined) => {
   }
 
   if('@container' in value) {
-    const container = value['@container'];
-    if(container !== '@list' && container !== '@set' &&
-      container !== '@index' && container !== '@language') {
-      let isValid = false;
-      const validContainers = ['@list', '@set', '@index', '@language'];
-      // JSON-LD 1.1 support
-      if(activeCtx['@version'] === 1.1) {
-        // || processingMode === 'jsonld-1.1') {
-        validContainers.push('@graph');
-        if(container === '@graph' ||
-          (_isArray(container) && container.length === 2 &&
-          container.includes('@graph') && container.includes('@set'))) {
-          isValid = true;
-        }
-      }
-      if(!isValid) {
-        throw new JsonLdError(
-          'Invalid JSON-LD syntax; @context @container value must be ' +
-          'one of the following: ' + validContainers.join(', '),
-          'jsonld.SyntaxError',
-          {code: 'invalid container mapping', context: localCtx});
-      }
+    // normalize container to an array form
+    const container = _isString(value['@container']) ? [value['@container']] : (value['@container'] || []);
+    const validContainers = ['@list', '@set', '@index', '@language'];
```

Comment:
This list is just static for 1.0 and 1.1 right?  May be more efficient to hoist it out to two static lists and choose which one to use based on mode at runtime.  Would avoid reallocation and updating per-call.

Reply:
Yes, that's what I do with my implementation, but I was just trying to follow the existing pattern. I can update to make these static arrays.

Reply:
Looking through the code, there's little uniformity for how such things are handled. Elsewhere in expand.js, there are inline arrays of keywords, and `api.isKeyword` is defined to use a switch statement to detect. Creating a top-level static array seems like yet a third way. Is a `switch` statement faster than `Array.includes`? Arguably, `Set.has` would be fast, if it worked consistently in `test-karma`. One could argue for a large refactor to unify the way all such lists are set and maintained.

Given that the `validContainers` is code I inherited, I'd rather not keep creeping this PR with things that should be done in a more general refactor.

Reply:
I agree that a larger refactor to make this stuff more consistent should be done in a separate PR.